vllm/worker/model_runner.py

```diff
@@ -69,6 +69,10 @@
 
 TModelInputForGPU = TypeVar('TModelInputForGPU', bound="ModelInputForGPU")
 
+# For now, bump up cache limits for recompilations during CUDA graph warmups.
+torch._dynamo.config.cache_size_limit = 128
+torch._dynamo.config.accumulated_cache_size_limit = 128
+
```

Comment:
I'm ok with this for now, but after I port the technique to skip dynamo guard evaluation overhead, this should not be necessary anymore.